<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlackJack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Player_Card_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>356</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_50">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>496</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Button_50{
background-color:  transparent;
border-image: url(:/resources/Images/Green_Chip.png);
background: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_100">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>496</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Button_100 {
background-color:  transparent;
border-image: url(:/resources/Images/Yellow_Chip.png);
background: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_10">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>496</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Button_10 {
background-color:  transparent;
border-image: url(:/resources/Images/Blue_Chip.png);
background: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Dra_Kort_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>496</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Dra Kort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stanna_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>496</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Stanna</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>346</y>
      <width>701</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 139, 139);
font: 36pt &quot;Segoe UI&quot;;
color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Spelarens Kort</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>46</y>
      <width>701</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 139, 139);
font: 36pt &quot;Segoe UI&quot;;
color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Datorns Kort</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Computer_Credits">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>222</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Credits">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>279</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>279</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>222</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>211</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 139, 139);
font: 18pt &quot;Segoe UI&quot;;
color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Bet:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 139, 139);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>266</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Current_Bet">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>211</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 139, 139);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 139, 139);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Text_Out">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>226</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Starta nytt spel i menyn</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>356</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>356</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>356</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>356</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>356</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>356</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(139, 139, 139);
font: 18pt &quot;Segoe UI&quot;;
color: rgb(225, 225, 225);</string>
       </property>
       <property name="text">
        <string> Datorn:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(139, 139, 139);
font: 18pt &quot;Segoe UI&quot;;
color: rgb(225, 225, 225);</string>
       </property>
       <property name="text">
        <string> Spelare:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Player_Card_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>326</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>326</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>326</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>326</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>56</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>56</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>56</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>56</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>86</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>86</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>86</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>56</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>56</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>86</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Comp_Card_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>56</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player_Card_Value">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Computer_Card_Value">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>460</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 139, 139);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 139, 139);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>16</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>501</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_16</zorder>
   <zorder>label_12</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>Player_Card_1</zorder>
   <zorder>Button_50</zorder>
   <zorder>Button_100</zorder>
   <zorder>Button_10</zorder>
   <zorder>Dra_Kort_button</zorder>
   <zorder>Stanna_button</zorder>
   <zorder>label_2</zorder>
   <zorder>label_7</zorder>
   <zorder>Player_Credits</zorder>
   <zorder>label_8</zorder>
   <zorder>Computer_Credits</zorder>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>label_9</zorder>
   <zorder>Current_Bet</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>Text_Out</zorder>
   <zorder>Player_Card_2</zorder>
   <zorder>Player_Card_3</zorder>
   <zorder>Player_Card_4</zorder>
   <zorder>Player_Card_5</zorder>
   <zorder>Player_Card_6</zorder>
   <zorder>Player_Card_7</zorder>
   <zorder>Player_Card_8</zorder>
   <zorder>Player_Card_9</zorder>
   <zorder>Player_Card_10</zorder>
   <zorder>Player_Card_11</zorder>
   <zorder>Comp_Card_5</zorder>
   <zorder>Comp_Card_2</zorder>
   <zorder>Comp_Card_3</zorder>
   <zorder>Comp_Card_1</zorder>
   <zorder>Comp_Card_9</zorder>
   <zorder>Comp_Card_8</zorder>
   <zorder>Comp_Card_11</zorder>
   <zorder>Comp_Card_6</zorder>
   <zorder>Comp_Card_4</zorder>
   <zorder>Comp_Card_10</zorder>
   <zorder>Comp_Card_7</zorder>
   <zorder>label_13</zorder>
   <zorder>label_17</zorder>
   <zorder>Computer_Card_Value</zorder>
   <zorder>label_18</zorder>
   <zorder>Player_Card_Value</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMeny">
    <property name="title">
     <string>Meny</string>
    </property>
    <addaction name="actionNytt_Spel"/>
   </widget>
   <addaction name="menuMeny"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNytt_Spel">
   <property name="text">
    <string>Nytt Spel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
